src/components/slider/Slider.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import style from './slider.module.scss';
+
+import NextIcon from '../icons/NextIcon';
+import PreviousIcon from '../icons/PreviousIcon';
+
+import img1Main from '../../assets/images/image-product-1.jpg';
+import img1Thumb from '../../assets/images/image-product-1-thumbnail.jpg';
+import img2Main from '../../assets/images/image-product-2.jpg';
+import img2Thumb from '../../assets/images/image-product-2-thumbnail.jpg';
+import img3Main from '../../assets/images/image-product-3.jpg';
+import img3Thumb from '../../assets/images/image-product-3-thumbnail.jpg';
+import img4Main from '../../assets/images/image-product-4.jpg';
+import img4Thumb from '../../assets/images/image-product-4-thumbnail.jpg';
+
+interface ImagePair {
+  main: string;
+  thumbnail: string;
+}
+interface SliderProps {
+  customArrowLeftClass?: string;
+  customArrowRightClass?: string;
+}
+
+const imagePair: ImagePair[] = [
+  { main: img1Main, thumbnail: img1Thumb },
+  { main: img2Main, thumbnail: img2Thumb },
+  { main: img3Main, thumbnail: img3Thumb },
+  { main: img4Main, thumbnail: img4Thumb },
+];
```

Comment:
It looks like images always will be the same. But what if you have another product in your app? With less/more images?